src/routers/alpha-router/alpha-router.ts

```diff
@@ -324,16 +325,17 @@ export type AlphaRouterParams = {
    */
   mixedSupported?: ChainId[];
 
-  /**
-   * The version of the universal router to use.
-   */
-  universalRouterVersion?: UniversalRouterVersion;
```

Comment:
not used anymore? curious why this was introduced originally?

Reply:
i introduced duplicate, at both [ctor](https://github.com/Uniswap/smart-order-router/blob/b3b8555a781a6373bf943a9cccc8696d121f4a8d/src/routers/alpha-router/alpha-router.ts#L330) level and method [param](https://github.com/Uniswap/smart-order-router/blob/b3b8555a781a6373bf943a9cccc8696d121f4a8d/src/routers/alpha-router/alpha-router.ts#L513) level. it's only needed at method param level, so im cleaning up in the ctor level